sdk/servicebus/service-bus/samples-dev/usingAadAuth.ts

```diff
@@ -7,15 +7,7 @@
  *
```

Comment:
I think we can just delete this one? all samples use AAD now anyway.

Reply:
removing